dataland-frontend/tests/e2e/specs/eu-taxonomy-financials/DataIntegrity.ts

```diff
@@ -50,6 +50,7 @@ describeIf(
       getKeycloakToken(uploader_name, uploader_pw).then((token: string) => {
         return uploadCompanyViaApi(token, generateDummyCompanyInformation(companyInformation.companyName)).then(
           (storedCompany): void => {
+            cy.ensureLoggedIn(uploader_name, uploader_pw);
             cy.visitAndCheckAppMount(
               `/companies/${storedCompany.companyId}/frameworks/${DataTypeEnum.EutaxonomyFinancials}/upload`
             );
```

Comment:
This comment is about the `pdfTest.pdf`` file.
Is it possible to move it to `./testing/data` ?.   Because there all our "fake" data lives and that is why it is where I first expected it when I started the review.

Reply:
Not for myself:
"pdfTest" should not be hardcoded anywhere in the code anymore!
It should be set once as "filename" for the dummy, and then be reused. Maybe use the Constants.ts?

Reply:
Centralized it in a Constants.ts for tests